### PR TITLE
Proofreading webpack.config.js for clarity

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| License       | MIT

Re-reading the default webpack.config.js file for clarity. I'm also updating the documentation at the moment, and will base those changes off of this.

### CODE REVIEW

File: symfony/webpack-encore-pack/1.0/webpack.config.js [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+    // (including one that's included on every page - e.g. "app")
+    .addEntry('app', './assets/js/app.js')
+    //.addEntry('page1', './assets/js/page1.js')
+    //.addEntry('page2', './assets/js/page2.js')
 
     .cleanupOutputBeforeBuild()
     .enableSourceMaps(!Encore.isProduction())
-
-    // the following line enables hashed filenames (e.g. app.abc123.css)
+    // enables hashed filenames (e.g. app.abc123.css)
```
Comment (Pierstoval) [inline symfony/webpack-encore-pack/1.0/webpack.config.js]: Maybe a hint telling that it's a good practice to use it conjointedly with `framework.assets.json_manifest_path`?

File: symfony/webpack-encore-pack/1.0/webpack.config.js [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
     .enableVersioning(Encore.isProduction())
 
-    // uncomment to define the assets of the project
-    //.addEntry('js/app', './assets/js/app.js')
-    //.addStyleEntry('css/app', './assets/css/app.scss')
```
Comment (Pierstoval) [inline symfony/webpack-encore-pack/1.0/webpack.config.js]: As you added a default JS entry, why not add one for CSS too?

**Comment by weaverryan** [comment]
Thanks for the reviews! I've got a WIP on this, so that I can create a corresponding PR to the docs so that everything is consistent (and we can merge at nearly the same time). I'll ping back when that's ready :)

File: symfony/webpack-encore-pack/1.0/webpack.config.js (hunk 2 above)
Comment (weaverryan) [reply symfony/webpack-encore-pack/1.0/webpack.config.js]: `addStyleEntry` is kind of an anti-pattern. So, I doc'ed it, but people shouldn't really need to use it, if they want to do things correctly.

**Comment by weaverryan** [comment]
This is ready for review! The big differences are that I created an `app.js` and `app.css` file with basic contents. The idea is that we recommend every app always start with these files, and these will be included in the layout. If it's an SPA or simple app, it's all they'll need. If they want page-specific JavaScript, then they'll add more entry files.

I've updated the docs with this change in mind: https://github.com/symfony/symfony-docs/pull/10128

**Pull request merged**